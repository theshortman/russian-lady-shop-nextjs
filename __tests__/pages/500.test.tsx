import { render } from "@testing-library/react";
import Page500 from "@/pages/500";

test("renders page 500 unchanged", () => {
  const { container } = render(<Page500 />);
  expect(container).toMatchSnapshot();
});
