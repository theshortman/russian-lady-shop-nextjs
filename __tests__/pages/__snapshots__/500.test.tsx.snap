// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders page 500 unchanged 1`] = `
<div>
  <div>
    <h1>
      Internal Server Error (#500)
    </h1>
    <p
      class="alert alert-danger"
    >
      Внутренняя ошибка сервера.
    </p>
    <p>
      Пожалуйста, 
      <a
        href="/contacts"
      >
        свяжитесь с нами
      </a>
      .
    </p>
  </div>
</div>
`;
